packages/conjure-client/src/fetchBridge/__tests__/fetchBridgeTests.ts

```diff
@@ -241,6 +241,30 @@ describe("FetchBridgeImpl", () => {
         await expect(bridge.callEndpoint(request)).resolves.toEqual(mockedResponseData);
     });
 
+    it("makes POST request with html text data", async () => {
+        const fakeHtmlData = "<html>Hello World</html>";
+        const request: IHttpEndpointOptions = {
+            data: fakeHtmlData,
```

Comment:
conjure payload is JSON, not HTML

Reply:
Thanks for the comment. That's true – and my primary reason for adding this `MediaType` is to support HTML as a response type, not a request type. I cribbed this test from the other `MediaType` tests, but can remove or modify it if you'd prefer.

Reply:
I'm not sure I understand. If Conjure (per conjure-spec) doesn't support text/html, then why do we need a test for it?

Reply:
Fair point! I wanted to use the `FetchBridge` from this library in a frontend API package, but the API in question is not actually written in Conjure, so I'll have to go back to the drawing board.